docs: Document Contact change log CSV export feature (#800)
mautic/user-documentation: docs/contacts/manage_contacts.rst

Some Plugins contain specific events. The events display and are searchable after installing and configuring the Plugin.

Exporting the change log

> ------------------------

Author: PR #15994 fixes a mislabeled CSV header in the Contact change log export (eventName → userSourceName). This documentation addition addresses the gap revealed by that fix - the export feature was previously undocumented. File: app/bundles/LeadBundle/Controller/AuditlogController.php

Source: https://github.com/mautic/mautic/pull/15994

> You can export the Contact's change log to CSV for offline analysis or record-keeping. The CSV includes columns for each event's timestamp, action, details, and the user or source that triggered the change.

Reviewer: ```suggestion
You can export the Contact's change log to CSV for offline analysis or record-keeping. The CSV includes columns for each event's timestamp, action, details, and the User or source that triggered the change.
```

> To export the change log, click the dropdown menu in the upper right corner of the Contact detail page and select the export option.

Reviewer: ```suggestion
.. vale off

To export the change log, click the dropdown menu in the upper right corner of the Contact detail page and select the export option.

.. vale on
```

Some Plugins contain specific events. The events display and are searchable after installing and configuring the Plugin.

Exporting the change log
------------------------

You can export the Contact's change log to CSV for offline analysis or record-keeping. The CSV includes columns for each event's timestamp, action, details, and the User or source that triggered the change.

.. vale off

> To export the change log, click the dropdown menu in the upper right corner of the Contact detail page and select the export option.

Reviewer: ```suggestion
To export the change log:

#. Click the down arrow icon to open dropdown menu in the upper right corner of the Contact detail page.
#. Select the export option.
```

Add this image:

<img width="2089" height="1173" alt="Image" src="https://github.com/user-attachments/assets/3955e7d7-01c5-4038-a329-4acee98d3b1c" />

Note for image:

- Use clear word to describe the image name. Use underscore to split words.
- Add descriptive alt text. Don't start with "Screenshot of" or anything alike.
